apps/code-examples/src/app/v2/v2.component.ts

```diff
@@ -16,7 +16,7 @@ import {
   SkyDocsShowcaseModule,
 } from '@skyux/docs-tools';
 import { SkyInputBoxModule } from '@skyux/forms';
-import { getDocumentationConfig } from '@skyux/manifest-dist';
+import { getDocumentationConfig } from '@skyux/manifest';
```

Comment:
The previous path doesn't actually work; modified the standard path in the tsconfig instead.